### ISSUE TITLE
update for new changes in Airbnb react native coding style

### ISSUE THREAD

**Issue by ikzjfr0** [tylerbuchea/badass-react-snippets#5]
Airbnb react native styles change in past 6 months, is there any plan to update?

**Comment by tylerthebuildor** [comment]
Definitely! I wasn't aware. But I'll look into the new styling and apply the changes as soon as possible. I still use this plugin often.

**Comment by tylerthebuildor** [comment]
@ikzjfr0 would love a pull request for this 😄  Also could you point me at the specific style documentation you're talking about?
